ftest-toxic-frontend-reporter/src/test/java/org/arquillian/cube/q/toxic/reporter/ToxicFunctionalTest.java

```diff
@@ -0,0 +1,76 @@
+package org.arquillian.cube.q.toxic.reporter;
+
+import org.arquillian.cube.HostIp;
+import org.arquillian.cube.impl.util.IOUtil;
+import org.arquillian.cube.q.api.NetworkChaos;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.arquillian.test.api.ArquillianResource;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+
+import java.io.IOException;
+import java.net.SocketException;
+import java.net.URL;
+
+import static org.arquillian.cube.q.api.NetworkChaos.DistributedLatencyType.logNormalLatencyInMillis;
+import static org.arquillian.cube.q.api.NetworkChaos.RateType.rate;
+import static org.arquillian.cube.q.api.NetworkChaos.TimeoutType.timeoutInMillis;
+import static org.arquillian.cube.q.api.Q.IterationRunCondition.times;
+
+@RunWith(Arquillian.class)
+public class ToxicFunctionalTest {
```

Comment:
Yes Matous we can create a new package, but it's always good to have functional tests in isolation.